### PR TITLE
Add rolebinding permission to rhods-dashboard

### PR DESCRIPTION
**Pull request by lucferbux**
- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-2856
- [x] The Jira story is acked
- [X] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

---

1. Install the build
2. Check that the settings sections is available.

Screenshot 2022-02-24 at 12 08 32

### CODE REVIEW

**Comment by crobby** [comment]
Is this a change that we'd also need upstream or just a downstream only thing?

**Comment by lucferbux** [comment]
> Is this a change that we'd also need upstream or just a downstream only thing?

Mmmm I'm not 100% sure, cause this allows to check if the user is admin to show this new section, but a part from the segment stuff they're planning to add more features, I'm not sure if this will go to upstream too.

**Comment by lucferbux** [comment]
> > Is this a change that we'd also need upstream or just a downstream only thing?
> 
> Mmmm I'm not 100% sure, cause this allows to check if the user is admin to show this new section, but a part from the segment stuff they're planning to add more features, I'm not sure if this will go to upstream too.

Now definitely needs to be bot upstream and downstream @crobby

File: odh-dashboard/base/cluster-role.yaml [hunk 1]
```diff
@@ -62,4 +62,11 @@ rules:
...
       - rhmis
-
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - user.openshift.io
+    resources:
+      - groups
```
Comment (anishasthana) [inline odh-dashboard/base/cluster-role.yaml:72]: Missing new line

**Review by anishasthana — APPROVED** [review]
We tested with http://quay.io/lferrnan/rhods-operator-live-catalog:1.8.3-segment-final. Changes work!

**Pull request merged**